NGRValidator/NGRValidator/Categories/NSData+NGRValidator.h

```diff
@@ -0,0 +1,23 @@
+//
+//  NSData+NGRValidator.h
+//  NGRValidator
+//
+//
+//
+
+#import "NGRMimeType.h"
+
+@interface NSData (NGRValidator)
+
+NS_ASSUME_NONNULL_BEGIN
+
+- (BOOL)ngr_hasMimeType:(NGRMimeType *)type;
+
+- (BOOL)ngr_isImage;
```

Comment:
This could be improved to conform to Swift naming guidelines in the following way:
`- (BOOL)ngr_isImage NS_SWIFT_NAME(isImage())` (or similar - not tested).